src/components/accordion/_accordion.scss

```diff
@@ -71,6 +71,9 @@
 
   .#{$prefix}--accordion__title {
     margin: $accordion-title-margin;
+    font-size: 1rem;
```

Comment:
Is there an easy way to keep this in sync with the `p` styles?

Reply:
Mix-in is something that comes to mind, but it raises a question of if we want to include all the styles of `<p>` or want to maintain a subset. Thoughts?

Reply:
I'm not sure why this would need to stay in sync with `p` styles?

Reply:
Good question @alisonjoseph! I think the talk was originally from the fact that this PR is creating an accordion header style that works both for `<p>` and `<div>`, and I copied a subset of `<p>` style here. Something that comes to my mind in this topic is https://twitter.com/marisafari/status/923571338765148161, but having brought it up, I think you made a good point that we just need to define a style that are not affected by future `<p>`/`<div>` style changes?

Reply:
Yeah, +1 to what @asudoh said. The node originally was a `p` tag and so it inherited `p` styles. Just want to make sure the type styles stay in sync with whatever they need to be, whether that's `p` or using our `typescale` mixin. Either way!

Reply:
👍 for `@include typescale()` - Made the change, thank you for the suggestion!